salt/orch/force_removal.sls

```diff
@@ -0,0 +1,72 @@
+# This is a best effort orchestration. Any operation run on the minions by this orchestration
+# could fail, as we don't know the current state of the cluster at this point. The idea is to
+# try to clean as much as possible where possible, but in the end the only thing we can really
+# ensure is that the salt-key of the target is rejected.
+
+# must provide the node (id) to be removed in the 'target' pillar
+{%- set target = salt['pillar.get']('target') %}
+{%- set target_nodename = salt.saltutil.runner('mine.get', tgt=target, fun='nodename')[target] %}
```

Comment:
You can use [get_nodename()](https://github.com/kubic-project/salt/blob/master/salt/_modules/caasp_net.py#L64), so it would look like:

```
{%- set target_nodename = salt.caasp_net.get_nodename(host=target) %}
```

Reply:
Thanks, will fix!

Reply:
This function doesn't work on the master because it got removed the case for `if __opts__['__role'] == 'master'` so it cannot be used when rendering as it is now.